patches/minecraft/net/minecraft/world/chunk/storage/RegionFile.java.patch

```diff
@@ -37,6 +37,6 @@
 +        return false;
 +    }
 +
+     @Nullable
+ 
      public synchronized DataInputStream func_76704_a(int p_76704_1_, int p_76704_2_)
-     {
-         if (this.func_76705_d(p_76704_1_, p_76704_2_))
```

Comment:
In this file, the earlier patch was applied in the wrong spot. The `@Nullable` was supposed to go above `func_76704_a` (weirdly with a new line between).
This patch puts it in the right place.